src/components/UniversalTable/UniversalTable.module.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../app/variables.scss";
+
+.TableContainer {
```

Comment:
why would you name your classes with a capital letter?

Reply:
solved